src/main/resources/portal.properties.EXAMPLE

```diff
@@ -231,10 +231,10 @@ oncoprint.hotspots.default=true
 # Custom gene sets
 # querypage.setsofgenes.location=file:/<path>
 
-# to enable use of Ehcache, set ehcache.cache_enabled to true and uncomment all Ehcache properties
-ehcache.cache_enabled=false
+# to enable use of Ehcache, set ehcache.cache_type to a valid cache type (heap, disk, hybrid)
+ehcache.cache_type=none
 #ehcache.xml_configuration=/ehcache.xml
-#ehcache.statistics_enabled=true
+#ehcache.statistics_enabled=false
 # Enable cache statistics endpoint for Ehcache monitoring
 #cache.statistics_endpoint_enabled=false
```

Comment:
there also seems to be some redundancy here - can you confirm if we need the property `cache.statistics_endpoint_enabled=false`

Reply:
yeah, this is for the actual cacheStats endpoint, the ehcache.statistics_enabled has to do with the actual configuration of the cacheManager. 

It's separate because ideally the cacheStats endpoint should be independent of whether or not we use Ehcache as our caching system.